pinot-core/src/main/java/org/apache/pinot/core/data/recordtransformer/DataTypeTransformer.java

```diff
@@ -104,4 +115,76 @@ public GenericRow transform(GenericRow record) {
     }
     return record;
   }
+
+  /**
+   * Standardize the value into supported types.
```

Comment:
Can we add more description on what this method does and what should be the expected behavior?

Reply:
Added